apis/fluent-en_GB/extensions/kotlin_1_3/atrium-api-fluent-en_GB-kotlin_1_3-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/kotlin_1_3/ResultFeatureAssertionsSpec.kt

```diff
@@ -0,0 +1,26 @@
+package ch.tutteli.atrium.api.fluent.en_GB.kotlin_1_3
+
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.specs.feature1
+import ch.tutteli.atrium.specs.fun2
+import ch.tutteli.atrium.specs.integration.ResultFeatureAssertionsSpec
+import ch.tutteli.atrium.specs.notImplemented
+
+class ResultFeatureAssertionsSpec : ResultFeatureAssertionsSpec(
+
+
+    feature1<Result<Int>, Int, Int>(Expect<Result<Int>>::isSuccess),
```

Comment:
the number after `feature` (or `fun`) defines the arity of the corresponding function or in other words, how many arguments it expects. `isSuccess` in this case does not expect any, thus you need to use `feature0` instead